src/main/scala/group/holmes/analytics/actors/AnalyticsSupervisor.scala

```diff
@@ -0,0 +1,28 @@
+package group.holmes.analytics.actors
```

Comment:
org.holmesprocessing is what we are using in Totem. Should we keep the structure or change it to match the LLC copywrite holder's name.